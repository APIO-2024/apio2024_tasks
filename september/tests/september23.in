txy4h26c1rm1uv8tr3eonahd67u8h56x
53
10 1
0 1 1 3 4 4 5 7 7
2 8 6 9 7 4 5 1 3
10 1
0 1 1 2 3 4 5 6 8
5 7 3 9 8 4 2 6 1
10 1
0 1 2 3 3 5 6 7 7
9 4 8 7 6 5 2 3 1
10 1
0 0 2 2 3 4 6 7 8
1 5 9 8 7 6 3 4 2
10 1
0 0 2 2 4 5 6 6 8
3 9 1 7 8 6 5 4 2
10 1
0 0 1 3 3 4 6 6 7
2 9 8 5 7 4 6 3 1
10 1
0 0 2 3 4 5 6 7 7
8 1 9 7 5 6 4 2 3
10 1
0 0 1 3 4 5 5 6 8
7 8 9 2 5 6 3 4 1
10 1
0 0 1 3 4 4 5 7 7
8 9 7 6 5 4 2 3 1
10 1
0 0 2 3 3 4 5 6 8
7 9 1 5 6 8 4 3 2
10 1
0 1 2 2 4 5 6 6 7
8 3 9 6 7 5 2 1 4
10 1
0 1 2 3 4 5 6 7 7
8 9 7 6 5 4 3 1 2
10 1
0 1 2 2 3 4 6 6 7
9 7 6 8 5 4 3 1 2
10 1
0 0 2 3 3 4 5 6 7
8 9 6 7 1 4 3 2 5
10 1
0 0 1 2 4 5 5 7 7
9 1 3 6 8 5 4 7 2
10 1
0 1 1 2 3 5 6 6 8
9 4 8 7 5 6 2 1 3
10 1
0 0 2 3 3 4 5 6 8
7 5 9 8 1 4 3 6 2
10 1
0 1 1 2 3 5 6 7 7
4 9 2 8 7 6 5 1 3
10 1
0 1 1 2 4 5 5 6 8
9 8 3 6 7 5 4 2 1
10 1
0 0 2 3 4 4 5 6 7
9 7 1 5 8 6 4 3 2
40 1
0 1 0 2 3 3 5 5 6 7 8 9 10 11 13 14 16 16 18 17 20 20 21 21 22 24 25 25 26 28 29 29 31 32 32 34 34 36 38
4 30 23 19 37 27 39 2 12 1 33 28 9 35 18 6 15 13 38 25 36 22 10 31 7 34 29 32 24 26 21 20 17 14 16 11 8 3 5
40 1
0 0 0 1 4 5 4 5 6 7 9 11 12 12 13 13 16 16 18 17 20 21 20 23 22 24 26 26 27 28 29 29 31 31 33 35 34 35 37
8 3 36 15 19 10 38 39 7 30 37 14 25 28 22 32 18 2 21 35 33 34 31 27 29 24 26 23 17 20 13 16 12 11 9 5 6 4 1
40 1
0 0 2 2 3 5 4 7 6 7 8 10 12 12 13 15 16 17 18 18 18 20 20 22 23 25 26 25 26 27 28 31 31 32 33 33 34 37 38
11 9 39 19 24 38 36 37 14 34 35 1 32 30 6 29 27 5 3 8 31 33 26 25 28 22 21 23 20 18 17 16 15 13 12 10 4 7 2
40 1
0 0 1 3 2 3 6 7 7 8 9 9 12 12 13 14 14 17 17 17 20 19 22 22 22 23 26 27 28 27 28 31 30 33 34 35 34 35 36
29 38 10 15 5 32 25 37 24 4 8 31 2 18 13 39 11 21 20 16 36 35 34 28 30 27 33 26 22 23 19 17 12 9 7 14 6 1 3
40 1
0 1 0 3 4 3 4 7 8 8 10 11 12 12 14 15 14 15 17 19 20 20 20 23 22 24 25 25 27 27 30 30 32 33 34 34 34 37 37
29 16 31 24 26 39 6 13 36 18 21 23 38 9 5 35 15 2 37 34 1 28 33 30 32 27 25 20 22 17 19 12 11 14 10 8 4 7 3
100 1
0 1 2 3 3 3 5 6 8 8 8 9 10 10 14 13 13 17 15 19 17 20 21 22 23 24 25 26 27 26 27 31 29 33 34 34 35 35 36 38 38 39 42 41 43 43 44 45 48 46 49 49 51 50 54 52 56 54 56 58 58 58 60 62 64 64 64 67 65 69 68 69 71 71 72 75 74 76 77 76 80 79 82 83 83 82 86 86 86 86 90 88 91 93 94 95 96 94 97
55 61 78 89 87 84 73 47 70 81 16 18 32 30 80 44 28 11 26 98 63 92 85 24 60 41 12 57 66 7 22 76 59 4 88 53 51 20 75 99 72 56 19 69 5 9 15 52 37 14 49 65 40 97 83 48 45 96 94 95 93 31 91 38 90 35 86 82 79 74 77 71 68 67 64 62 58 54 50 46 43 42 39 34 36 33 29 27 25 23 17 21 13 10 8 6 3 2 1
1000 1
0 1 1 2 3 2 5 5 4 8 6 11 7 9 11 15 15 17 13 16 18 17 21 21 23 21 25 26 27 25 30 27 28 33 33 34 33 34 34 39 39 41 41 40 44 45 44 46 46 47 50 51 52 51 52 52 52 56 54 54 58 60 62 60 59 63 65 62 64 65 67 70 69 68 70 75 73 72 78 79 80 77 78 80 79 84 85 83 86 88 88 86 90 88 94 92 94 94 95 95 100 99 102 99 102 103 101 105 108 105 110 106 108 108 114 113 115 117 116 116 118 120 119 119 119 122 125 126 125 124 126 129 132 132 131 131 133 133 137 137 136 136 141 139 141 143 142 142 146 148 145 151 151 148 152 154 154 153 157 156 157 157 162 159 161 161 161 167 164 164 166 166 168 173 169 171 174 175 178 175 175 178 179 180 180 185 184 183 184 184 188 186 190 188 194 192 196 196 196 196 196 197 200 200 203 202 206 207 203 208 207 207 209 209 214 212 213 216 215 217 219 220 221 220 221 222 223 222 223 224 226 227 229 232 231 235 232 237 237 238 236 237 242 238 239 244 244 245 243 248 249 247 251 251 253 250 253 252 254 256 260 260 258 259 264 260 263 264 263 267 266 267 268 268 274 270 273 272 274 275 275 281 281 278 280 284 286 285 288 284 287 288 287 288 291 295 293 297 296 294 297 300 300 299 299 303 302 302 305 306 309 310 310 309 310 315 315 313 318 316 318 318 322 321 322 323 323 327 327 327 329 329 332 330 334 335 334 332 337 336 337 338 337 342 341 342 345 345 344 349 349 351 350 352 351 351 355 356 358 358 360 361 361 361 363 362 362 364 364 369 370 367 369 368 373 374 374 375 373 378 375 376 380 381 381 382 384 387 384 385 387 389 392 390 391 391 394 394 398 397 397 397 402 402 401 403 402 405 405 405 407 406 412 409 411 414 411 412 414 418 420 421 419 420 419 423 426 424 423 428 428 431 430 430 430 432 431 434 437 434 439 437 439 443 444 441 443 442 445 444 448 451 450 453 453 453 451 452 457 455 458 459 461 459 460 461 464 462 465 467 466 469 467 469 474 472 471 477 475 475 476 477 481 483 484 485 485 485 485 488 486 489 490 492 489 491 493 492 495 497 499 498 497 503 499 505 504 503 508 508 508 507 511 513 511 511 513 514 513 516 515 519 518 518 522 525 525 523 527 529 525 529 530 530 532 535 532 532 533 536 539 541 537 540 540 544 546 546 545 546 548 546 551 551 554 551 553 556 557 557 560 561 562 560 564 565 566 565 566 569 566 566 568 572 573 574 574 574 578 575 577 576 581 582 580 583 585 582 585 587 588 588 590 593 591 595 594 595 593 597 595 596 598 602 601 605 604 602 604 606 606 606 607 612 614 610 611 615 613 616 620 621 620 619 624 625 624 626 623 624 629 627 628 632 632 635 633 635 634 639 638 638 642 639 641 643 645 644 644 646 650 649 650 650 649 651 651 655 655 659 657 657 661 661 661 662 665 666 664 664 666 667 669 672 670 673 671 675 678 676 676 680 677 680 681 680 685 685 684 688 687 689 687 690 690 690 695 696 693 695 695 697 700 701 701 700 702 703 708 707 705 711 707 713 710 714 716 712 717 717 719 721 720 722 724 722 722 723 723 729 726 727 728 728 734 733 736 732 736 734 738 738 742 741 739 745 746 743 748 747 749 748 749 752 749 750 755 756 755 758 758 758 757 760 762 763 763 766 764 769 770 770 768 769 769 775 773 776 774 775 780 781 777 781 779 780 782 784 785 788 790 786 787 791 790 793 795 793 793 795 797 801 797 798 802 803 805 806 804 805 806 811 810 810 813 814 816 815 816 819 817 819 818 820 821 825 823 826 825 826 828 826 828 833 830 831 833 837 834 834 836 837 838 838 841 844 845 846 848 847 845 850 852 852 852 853 855 856 857 856 857 858 859 860 860 863 861 863 867 867 870 869 872 868 874 870 874 872 875 877 877 877 879 881 883 883 884 882 885 886 888 889 890 891 892 894 895 895 894 895 898 901 898 903 904 901 906 905 907 906 910 907 908 909 909 915 916 914 916 915 920 919 917 919 921 923 921 924 928 927 928 926 930 929 930 934 933 932 935 939 936 939 940 938 943 942 941 945 945 945 947 947 950 949 951 954 954 953 958 955 959 959 957 962 963 962 962 967 965 964 965 968 969 973 971 973 971 975 975 979 978 977 979 978 982 983 985 982 983 988 987 986 987 992 989 992 992 995 995
922 111 636 91 899 463 87 571 900 348 279 440 195 706 772 783 822 991 261 339 981 282 228 218 353 709 216 948 57 37 735 24 839 425 144 328 97 49 552 422 538 691 865 298 998 617 417 937 674 715 324 622 427 880 346 710 999 212 29 683 85 138 824 851 121 913 36 842 357 433 53 570 478 911 304 925 61 956 399 371 301 677 547 181 135 652 307 447 740 343 255 586 123 521 730 22 550 974 871 952 189 671 789 908 110 902 526 961 14 321 603 283 794 807 976 973 170 482 416 559 496 731 81 608 240 692 487 10 426 777 276 534 660 265 584 311 317 637 912 933 320 621 140 778 201 396 500 8 849 130 744 812 491 204 98 918 134 42 398 71 225 233 290 262 864 281 277 689 383 354 609 984 860 542 598 663 995 653 211 19 470 993 829 13 583 446 873 558 887 563 659 567 897 737 792 569 647 771 884 896 20 656 754 379 163 127 969 785 531 319 191 297 377 388 74 776 355 292 581 972 741 645 234 753 158 726 658 273 257 205 176 592 672 316 905 589 128 502 308 436 155 878 246 835 380 182 830 827 611 55 684 579 477 679 517 96 32 840 372 302 271 996 699 678 765 139 693 312 944 862 109 378 253 408 350 89 471 577 599 395 515 718 725 729 600 791 333 914 366 631 704 269 9 331 786 479 799 230 910 391 162 107 629 938 648 352 149 687 224 367 92 712 66 994 152 410 150 509 187 779 997 843 76 980 35 31 370 421 854 325 27 562 866 501 12 809 561 989 881 441 946 289 4 752 124 876 38 498 904 932 347 171 326 153 675 345 415 7 199 16 555 820 82 894 708 970 633 30 655 524 668 520 229 640 494 848 147 512 823 165 400 293 630 549 315 770 978 686 649 960 435 365 992 942 151 270 773 694 456 872 480 724 804 834 475 682 393 734 761 145 194 964 711 703 68 990 5 468 137 818 506 387 966 93 623 667 759 543 870 811 505 462 404 449 63 43 903 507 432 556 893 25 545 698 363 618 118 41 160 808 858 644 474 362 86 597 594 700 314 641 554 819 3 62 768 112 955 504 188 454 815 411 466 23 813 359 620 438 413 832 340 101 665 856 210 537 341 473 846 680 266 499 800 869 58 844 495 84 77 628 486 926 986 616 838 853 90 67 492 198 931 696 510 407 626 309 751 434 117 156 968 516 48 615 767 73 890 337 305 80 610 625 923 837 541 69 796 967 917 104 886 528 177 774 951 172 798 891 430 523 299 75 793 296 705 766 429 386 79 392 867 336 888 208 670 241 983 423 861 467 193 419 183 146 295 762 882 701 445 654 833 179 335 133 414 106 143 141 763 115 166 757 464 260 459 593 64 178 114 382 457 100 590 291 578 988 60 806 287 987 103 979 614 56 190 518 46 947 45 184 666 174 612 962 985 877 409 334 982 959 787 606 975 782 941 639 330 634 489 587 585 376 52 461 580 256 539 389 605 458 236 601 173 936 168 958 452 953 934 514 803 250 662 977 971 575 132 286 248 384 167 284 755 129 573 405 278 125 207 568 929 374 928 206 924 965 749 401 272 235 202 963 451 231 368 245 197 226 119 239 533 448 267 957 919 530 222 954 748 743 950 742 949 161 945 263 220 738 442 916 217 732 258 727 213 252 247 721 722 719 244 238 940 943 939 935 930 927 921 920 915 907 909 906 901 895 892 898 889 885 883 879 875 874 868 863 859 855 857 852 850 847 845 841 836 831 828 826 825 821 817 814 816 810 802 805 801 797 795 790 784 788 780 781 775 764 769 760 758 756 750 747 746 745 739 736 733 728 720 723 717 716 713 714 702 707 697 695 690 688 685 681 676 673 669 664 661 657 651 650 646 643 642 638 635 632 627 619 624 613 607 602 604 596 595 591 588 582 576 574 572 566 565 564 560 553 557 551 546 548 544 536 540 535 529 532 527 522 525 519 513 511 508 503 497 493 490 488 485 484 481 483 476 469 472 465 460 455 453 450 443 444 439 437 431 428 424 420 418 406 412 403 402 397 390 394 385 381 375 369 373 361 364 360 358 356 351 349 344 342 338 332 327 329 323 318 322 313 306 310 303 300 294 288 280 285 275 274 268 259 264 254 249 251 243 237 242 232 223 227 221 215 219 214 209 203 200 196 192 185 186 180 175 169 164 159 157 154 148 142 136 131 126 122 120 116 108 113 102 105 99 95 88 94 83 78 72 70 65 59 54 51 50 47 44 40 39 33 34 28 26 21 18 17 15 11 6 2 1
100 1
0 1 2 2 4 5 3 6 7 7 10 11 12 10 12 13 15 16 15 17 18 18 20 23 23 25 23 25 25 28 29 31 30 33 34 35 34 35 37 39 37 41 42 42 43 42 45 46 48 47 47 49 49 52 51 53 56 55 56 59 57 61 62 60 62 64 63 64 66 67 68 71 71 70 73 74 76 76 75 77 77 80 82 83 84 85 84 86 85 89 90 89 91 91 93 93 95 96 97
88 8 87 81 21 78 36 44 72 94 86 50 22 54 99 79 18 6 16 19 9 40 32 24 97 13 65 38 69 58 31 92 29 26 95 52 35 27 14 66 5 75 4 98 96 93 39 55 73 71 68 91 90 51 47 89 60 64 45 85 59 43 84 83 82 80 77 76 74 67 70 63 62 57 61 53 56 49 48 46 42 37 41 34 33 30 28 25 23 20 17 15 12 11 10 3 7 2 1
40 1
0 0 0 1 4 3 5 7 6 9 9 11 10 12 12 14 14 17 18 17 19 20 21 23 24 23 26 26 27 28 30 30 30 33 34 34 36 36 37
15 25 16 35 31 39 32 37 2 13 29 10 27 24 8 36 22 38 7 20 34 33 5 4 1 28 26 23 30 21 18 19 17 14 11 12 6 3 9
40 1
0 0 2 3 3 3 6 6 7 7 10 11 11 13 13 15 16 16 16 18 19 20 22 22 22 23 24 26 27 27 28 31 31 32 32 34 35 35 38
8 25 14 36 4 12 34 5 37 30 21 33 29 19 27 24 9 1 39 17 38 35 32 31 28 23 22 26 18 16 15 20 11 13 10 7 3 6 2
40 1
0 1 2 3 4 4 6 7 7 7 10 10 11 11 14 14 16 16 17 17 20 21 21 21 24 24 24 27 28 27 29 31 30 31 32 35 36 36 38
25 9 23 8 34 5 33 15 13 37 22 26 18 39 19 30 12 36 35 38 31 32 28 29 24 27 21 17 16 20 14 11 10 6 7 4 3 1 2
40 1
0 1 2 1 3 4 6 7 7 9 9 9 10 13 14 13 14 17 18 17 20 21 20 22 24 24 25 26 28 27 29 29 31 31 32 33 35 35 38
37 11 36 23 15 39 33 16 34 38 8 12 35 19 30 31 27 5 3 32 25 18 2 28 26 24 29 22 20 21 17 14 13 10 9 6 7 4 1
40 1
0 1 2 3 3 5 5 7 8 8 8 11 10 12 14 15 15 17 17 19 19 20 22 21 24 23 25 27 26 27 29 29 31 32 33 34 35 35 36
28 30 16 13 37 18 10 4 39 27 25 38 36 6 34 9 35 33 24 32 21 31 29 26 23 20 22 19 17 15 14 12 11 7 8 5 3 1 2
10 1
0 0 1 3 3 4 5 6 8
2 8 9 4 7 5 6 1 3
10 1
0 1 2 2 3 4 5 7 8
9 6 4 7 8 5 3 1 2
10 1
0 0 2 2 3 4 6 7 8
1 9 5 8 3 6 7 2 4
10 1
0 0 2 3 3 4 6 6 7
5 1 9 8 7 4 6 3 2
10 1
0 1 1 2 4 5 6 6 7
9 8 7 3 6 4 2 1 5
10 1
0 0 1 2 3 4 6 7 8
8 9 5 3 1 7 6 4 2
10 1
0 1 2 3 4 5 6 6 8
9 7 8 6 5 3 4 2 1
10 1
0 1 2 3 3 4 5 6 8
9 7 8 5 4 6 3 1 2
10 1
0 0 2 3 4 5 6 6 7
8 9 7 1 6 5 4 3 2
10 1
0 1 1 2 3 5 6 6 8
4 2 7 9 8 6 3 5 1
10 1
0 0 2 2 4 5 5 6 7
8 9 1 7 5 3 6 4 2
10 1
0 0 2 2 4 4 6 6 7
3 5 8 9 1 7 4 6 2
10 1
0 1 2 3 3 5 6 7 7
9 8 7 6 4 5 2 3 1
10 1
0 0 1 3 3 4 5 6 7
9 8 7 6 4 5 2 3 1
10 1
0 0 1 2 4 5 6 6 7
9 3 8 1 7 6 5 2 4
10 1
0 1 1 3 3 5 6 7 7
9 2 8 7 6 5 4 3 1
10 1
0 1 1 2 4 4 5 6 8
9 8 7 3 6 5 4 1 2
10 1
0 1 2 2 4 5 5 7 8
3 9 8 7 6 5 4 1 2
10 1
0 1 2 2 3 5 5 6 7
4 8 9 7 6 5 2 3 1
10 1
0 1 1 2 3 4 5 6 7
8 9 7 3 5 6 4 2 1
