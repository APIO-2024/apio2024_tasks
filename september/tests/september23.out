7ckgnn4wyi495puj3ibqf81dqvapyv6b
7
6
8
9
9
8
7
6
9
8
6
8
7
7
6
7
7
8
9
9
35
34
36
32
32
95
957
93
30
32
30
34
36
4
7
7
8
6
8
8
7
9
8
7
8
8
9
8
9
8
8
8
8
